rust/kcl-lib/src/std/array.rs

```diff
@@ -280,33 +149,13 @@ pub async fn push(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, K
         RuntimeType::any()
     };
 
-    let new_array = inner_push(values, item);
+    let new_array = inner_push(values, item).await;
 
     Ok(KclValue::HomArray { value: new_array, ty })
 }
 
-/// Append an element to the end of an array.
-///
-/// Returns a new array with the element appended.
-///
-/// ```no_run
-/// arr = [1, 2, 3]
-/// new_arr = push(arr, item = 4)
-/// assert(new_arr[3], isEqualTo = 4, tolerance = 0.1, error = "4 was added to the end of the array")
-/// ```
-#[stdlib {
-    name = "push",
-    keywords = true,
-    unlabeled_first = true,
-    args = {
-        array = { docs = "The array which you're adding a new item to." },
-        item = { docs = "The new item to add to the array" },
-    },
-    tags = ["array"]
-}]
-fn inner_push(mut array: Vec<KclValue>, item: KclValue) -> Vec<KclValue> {
+async fn inner_push(mut array: Vec<KclValue>, item: KclValue) -> Vec<KclValue> {
```

Comment:
Just wondering, is there a reason these got changed back to async? I thought I was cleaning stuff up by removing it in my last PR.

Reply:
Sorry, rebasing error, fixed now